config/initializers/constants.rb

```diff
@@ -1,3 +1,9 @@
+# ECF version 1 released: 1st September 2021
+ECF_ROLLOUT_DATE = Date.new(2021, 9, 1).freeze
+
+# Introduction of ECT statutory induction: 1st September 1999
+STATUTORY_INDUCTION_ROLLOUT_DATE = Date.new(1999, 9, 1).freeze
```

Comment:
Think we'll need these values outside of the realm of induction period validation?